app/_data/docs_nav_kic_2.9.x.yml

```diff
@@ -115,6 +113,8 @@ items:
       - text: Exposing a UDP Service
         url: /guides/using-udpingress
         generate: false # Generated in the CustomResources section above
+      - text: Exposing a gRPC service
+        url: /guides/using-ingress-with-grpc
```

Comment:
I like this. Even more so, I was thinking about putting those 3: `Exposing a TCP|UDP|gRPC Service` under a common nav section like `Exposing services` so that the navigation bar panel on the left gets cleaned up a bit. But that can be done on a separate occassion.

Reply:
yeah, it makes total sense. Let me see how much effort it requires

Reply:
I've set the 3 under a common section, PTAL